WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -104,4 +110,27 @@ private extension SiteStatsPeriodViewModel {
         return dataRows
     }
 
+    func searchTermsTableRows() -> [ImmuTableRow] {
+        var tableRows = [ImmuTableRow]()
+        tableRows.append(CellHeaderRow(title: PeriodHeaders.searchTerms))
+        tableRows.append(TopTotalsPeriodStatsRow(itemSubtitle: SearchTerms.itemSubtitle,
+                                                 dataSubtitle: SearchTerms.dataSubtitle,
+                                                 dataRows: searchTermsDataRows(),
+                                                 siteStatsPeriodDelegate: periodDelegate))
+
+        return tableRows
+    }
+
+    func searchTermsDataRows() -> [StatsTotalRowData] {
+        let searchTerms = store.getTopSearchTerms()
+        var dataRows = [StatsTotalRowData]()
+
+        searchTerms?.forEach { item in
```

Comment:
nitpick: this whole method is is basically a one-line map:
```swift
return  store.getTopSearchTerms().map { StatsTotalDataRow(name: item.label, data: item.value.displayString()) }
```

I personally find the `map` version much easier to read, but up to you :)